docs/sources/tempo/operations/tempo_cli.md

```diff
@@ -99,6 +99,46 @@ Options:
 Streaming over HTTP requires the `stream_over_http_enabled` flag to be set. For more information, refer to [Tempo GRPC API documentation]({{< relref "../api_docs" >}}).
 {{% /admonition %}}
 
+### Search tags
+Call the Tempo API and search attribute names.
+
+```bash
+tempo-cli query api search-tags <host-port> [<start> <end>]
+```
+Arguments:
+- `host-port` A host/port combination for Tempo. The scheme will be inferred based on the options provided.
+- `start` Start of the time range to search: (YYYY-MM-DDThh:mm:ss)
+- `end` End of the time range to search: (YYYY-MM-DDThh:mm:ss)
+
+Options:
+- `--org-id <value>`      Organization ID (for use in multi-tenant setup).
+- `--use-grpc`            Use GRPC streaming
+- `--path-prefix <value>` String to prefix search paths with
+
+{{% admonition type="note" %}}
+Streaming over HTTP requires the `stream_over_http_enabled` flag to be set. For more information, refer to [Tempo GRPC API documentation]({{< relref "../api_docs" >}}).
+{{% /admonition %}}
+
+### Search tag values
+Call the Tempo API and search attribute values.
+
+```bash
+tempo-cli query api search-tag-values <tag> <host-port> [<start> <end>]
+```
+Arguments:
+- `host-port` A host/port combination for Tempo. The scheme will be inferred based on the options provided.
+- `tag` The fully qualified traceql tag to search for. e.g. `resource.service.name`
+- `start` Start of the time range to search: (YYYY-MM-DDThh:mm:ss)
+- `end` End of the time range to search: (YYYY-MM-DDThh:mm:ss)
+
+Options:
+- `--org-id <value>`      Organization ID (for use in multi-tenant setup).
+- `--use-grpc`            Use GRPC streaming
+- `--path-prefix <value>` String to prefix search paths with
```

Comment:
This is the kind of thing you might consider using a [definition list](https://www.markdownguide.org/extended-syntax/#definition-lists), which is supported by Hugo. That the top item (the term) is bold, and the definition appears underneath. 

```suggestion
Arguments:
`host-port` 
: A host/port combination for Tempo. The scheme will be inferred based on the options provided.

`tag` 
: The fully qualified traceql tag to search for; for example: `resource.service.name`

`start`
: Start of the time range to search: (`YYYY-MM-DDThh:mm:ss`)

`end`
: End of the time range to search: (`YYYY-MM-DDThh:mm:ss`)

Options:
`--org-id <value>`
: Organization ID (for use in multi-tenant setup).

`--use-grpc`
: Use GRPC streaming

`--path-prefix <value>`
: String to prefix search paths with
```